src/components/CitationExporter/components/SortSelector.tsx

```diff
@@ -0,0 +1,13 @@
+import { SolrSort } from '@api';
+import { Sender } from '@xstate/react/lib/types';
+import { CitationExporterEvent } from '../CitationExporter.machine';
+
+export const SortSelector = (props: { sort: SolrSort[]; dispatch: Sender<CitationExporterEvent> }) => {
+  const { dispatch, sort = ['date desc'] } = props;
+  const handleSortChange = (newSort: SolrSort[]) => {
+    dispatch({ type: 'SET_SORT', payload: newSort });
+  };
+
+  return <div></div>;
```

Comment:
Intentionally not shown?

Reply:
Yep, although this one can be shown now that the `fullWidth` param is in (I'll rebase before merge)